### PR TITLE
Wind demo: update using newest luma.gl TF API

### PR DESCRIPTION
**Pull request by 1chandu**
- REQUIRES luma.gl 5.1.0.
- Update wind demo to use newer luma.gl TransformFeedback API.
- Use instanced rendering for particle layer.

Verified running wind demo using latest luma.gl build.

### CODE REVIEW

File: examples/wind/src/layers/particle-layer/particle-layer.js [hunk 1]
```diff
@@ -329,7 +325,7 @@ export default class ParticleLayer extends Layer {
     // This will be a grid of elements
     this.state.numInstances = nx * ny;
 
-    const positions3 = this.calculatePositions3({nx, ny});
```
Comment (ibgreen) [inline examples/wind/src/layers/particle-layer/particle-layer.js]: ? Don't you need this grid?

File: examples/wind/src/layers/particle-layer/particle-layer.js [hunk 2]
```diff
@@ -167,7 +167,7 @@ export default class ParticleLayer extends Layer {
     });
 
     model.setAttributes({
-      posFrom: bufferTo
+      posFrom: bufferTo.setDataLayout({size: 4, instanced: 1})
```
Comment (ibgreen) [inline examples/wind/src/layers/particle-layer/particle-layer.js]: Q: Are we constantly resetting the layout to the same values? can this done during init?

Comment (1chandu) [reply examples/wind/src/layers/particle-layer/particle-layer.js]: We are changing layout between TF and regular runs. In next luma PR I am actually adding a new method to update layout, which is lightweight than `setDataLayout`.

File: examples/wind/src/layers/particle-layer/particle-layer.js (hunk 1 above)
Comment (1chandu) [reply examples/wind/src/layers/particle-layer/particle-layer.js]: Now I am using instanced rendering, so I am just rendering a sing instanced point and actual grid offsets are coming from a different instanced buffer (output of TransformFeedback).

**Pull request merged**